### DISCUSSION TITLE
How can I deploy a self-hosted MyScaleDB on K8s?

### DISCUSSION THREAD

**Discussion by dragonborn-lxc** [myscale/myscaledb#43] [Q&A]
Does it support deploying with Helm?
And how much data does self-hosted support?
Thanks :)

**Comment by lqhl** [comment]
MyScaleDB can be deployed on Kubernetes using the [ClickHouse Operator](https://github.com/Altinity/clickhouse-operator). We plan to draft documentation on this in the coming weeks.

MyScaleDB supports ClickHouse's distributed table engine, allowing for theoretical scalability to handle billion-scale vectors. We recommend allocating 32GB of RAM for approximately 5 million 768-dimensional vectors when using the ScaNN vector index.
